bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -1433,6 +1433,18 @@ private function addWorkflowNode(ArrayNodeDefinition $rootNode)
                                     ->defaultNull()
```

Comment:
please set this option to deprecated.

Reply:
Ah, didn't even know that was an option. If Travis is happy, I think I've resolved both your comments. Thanks for the review!